python/ray/util/multiprocessing/pool.py

```diff
@@ -156,6 +157,14 @@ def __init__(self, underlying):
 
 
 class ResultThread(threading.Thread):
+    # Thread that collects results from distributed actors.
```

Comment:
Great comment! Can we convert it to this docstring format:

```python
class ResultThread(threading.Thread):
"""Thread that collects results from distributed actors.

It winds down when either:
    - A pre-specified number of objects has been processed
    - When the END_SENTINEL (submitted through self.add_object_ref())
         has been received and all objects received before that have been
         processed.

Initialize the thread with total_object_refs = float('inf') to wait for the
END_SENTINEL.

Args:
    object_refs (List[RayActorObjectRefs]): ObjectRefs to Ray Actor calls.
        Thread tracks whether they are ready. More ObjectRefs may be added
        with add_object_ref (or _add_object_ref internally).
    single_result (bool): Should be True if the thread is managing function
        with a single result (like apply_async). False if the thread is managing
        a function with a List of results.
    callback (Callable): called only once at the end of the thread
        if no results were errors. If single_result=True, and result is
        not an error, callback is invoked with the result as the only
        argument. If single_result=False, callback is invoked with
        a list of all the results as the only argument.
    error_callback (Callable): called only once on the first result
        that errors. Should take an Exception as the only argument.
        If no result errors, this callback is not called.
    total_object_refs (int): Number of ObjectRefs that this thread
        expects to be ready. May be more than len(object_refs) since
        more ObjectRefs can be submitted after the thread starts.
        If None, defaults to len(object_refs). If float("inf"), thread runs
        until END_SENTINEL (submitted through self.add_object_ref())
        has been received and all objects received before that have
        been processed.
"""
```

Reply:
Good suggestion. Perhaps one more addition to the explanation of `object_refs`: 
```
    object_refs (List[RayActorObjectRefs]): ObjectRefs to Ray Actor calls.
        Thread tracks whether they are ready. More ObjectRefs may be added
        with add_object_ref (or _add_object_ref internally) until the total object
        count is total_object_refs.
```

Reply:
Good idea, let's add it!

Reply:
done!